app/controllers/aspects_controller.rb

```diff
@@ -51,22 +49,20 @@ def destroy
   end
 
   def show
-    if @aspect = current_user.aspects.where(:id => params[:id]).first
-      redirect_to aspects_path('a_ids[]' => @aspect.id)
+    if aspect
+      redirect_to aspects_path('a_ids[]' => aspect.id)
     else
       redirect_to aspects_path
     end
   end
 
   def update
-    @aspect = current_user.aspects.where(:id => params[:id]).first
-
-    if @aspect.update_attributes!(aspect_params)
-      flash[:notice] = I18n.t 'aspects.update.success', :name => @aspect.name
+    if aspect.update_attributes!(aspect_params)
+      flash[:notice] = I18n.t 'aspects.update.success', :name => aspect.name
     else
-      flash[:error] = I18n.t 'aspects.update.failure', :name => @aspect.name
+      flash[:error] = I18n.t 'aspects.update.failure', :name => aspect.name
```

Comment:
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.